app/uk/gov/hmrc/testuser/services/Generator.scala

```diff
@@ -213,3 +214,38 @@ class PensionSchemeAdministratorIdentifierGenerator(random: Random = new Random)
     PensionSchemeAdministratorIdentifier(s"$initialCharacter$remainingDigits")
   }
 }
+
+object VrnChecksum {
```

Comment:
This code isn't tested.

Reply:
It is tested in GeneratorSpec

Reply:
In GeneratorSpec, the test verifies that the generated VRN can be passed to VrnCheckSum.isValid and result in a return value of true.

What would happen if isValid was changed to:

def isValid(vrn: String) = true

I think that all the tests would pass because there are no tests that verify the logic of actually calculating the checksum.  I would expect to see some tests that verified that VRNs with valid and invalid checksums returned the correct result from isValid.

Reply:
it has been added in the last commit